cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -212,6 +218,7 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
                 config.analyzer.sw360Configuration?.let {
                     Sw360PackageCurationProvider(it).takeIf { useSw360Curations }
                 },
+                OrtConfigPackageCurationProvider().takeIf { useOrtCurations },
```

Comment:
What's the rational for using the curations only as a fallback, instead of merging these into `packageCurations.toSet()`?

Reply:
Simply because that is the existing mechanism for the other providers and I did not want to change that now. I agree that it should not be only a fallback, but the same argument applies to the other providers and I'd rather discuss this with the team first before introducing a new mechanism.

Reply:
Would it make sense to provide that rationale in the commit message?

Reply:
FYI, I have added the fallback topic to the agenda of the next dev meeting.